pyproject.toml

```diff
@@ -182,6 +182,7 @@ hiredis = { version = "*", optional = true }
 Pympler = { version = "*", optional = true }
 parameterized = { version = ">=0.7.4", optional = true }
 idna = { version = ">=2.5", optional = true }
+rust-python-jaeger-reporter = {git = "https://github.com/erikjohnston/rust-jaeger-python-client.git"}
```

Comment:
`rust-python-jaeger-reporter` ["is not configurable and is hardcoded to report to the local agent on localhost and the default port."](https://github.com/erikjohnston/rust-jaeger-python-client/tree/27754b5f56b4099374c389f677911f7af852a00e#limitations). 
 
(╯°□°）╯︵ ┻━┻ time to use OpenTelemetry stuff https://github.com/matrix-org/synapse/issues/11850

Reply:
-> https://github.com/matrix-org/synapse/pull/13400

Reply:
Updated `rust-python-jaeger-reporter` to make the `host` and `port` configurable via https://github.com/erikjohnston/rust-jaeger-python-client/pull/8